app/models/link_checker_api_report/link.rb

```diff
@@ -4,6 +4,10 @@ class LinkCheckerApiReport::Link < ApplicationRecord
 
   belongs_to :report, class_name: "LinkCheckerApiReport"
 
+  scope :status_ok, -> { where(status: "ok") }
+  scope :created_six_months_ago, -> { where("created_at < ?", 6.months.ago) }
```

Comment:
Is six months enough? I'd go even shorter: three.

Reply:
I wanted to be cautious at the beginning to see what sort of space savings we get with six months. We could certainly reduce it to three in the future if we feel that six months is too long.